src/featureModals/Beacon/BeaconAuthorization.tsx

```diff
@@ -87,8 +87,8 @@ const BeaconAuthorize = ({ open, managerBalance, onClose }: Props) => {
     const { destination, amount, parameters } = operationDetails[0];
     const operationParameters = parameters || { value: { prim: 'Unit' }, entrypoint: 'default' };
 
-    const estimatedMinimumFee = estimateOperationGroupFee(selectedParentHash, operationDetails).estimatedFee;
     const estimates = estimateOperationGroupFee(selectedParentHash, operationDetails);
+    const estimatedMinimumFee = estimates.estimatedFee;
```

Comment:
async nature would not allow for the variable to be assigned correctly m we'd have to use useEffect if want to retain estimatedMinimumFee